examples/end_to_end_tutorial.ipynb

```diff
@@ -3839,9 +3836,9 @@
  "metadata": {
   "hide_input": false,
   "kernelspec": {
-   "display_name": "namematch",
+   "display_name": "nm_prod_3.8",
```

Comment:
Just noting the kernel change here. This is fine though since we can't expect people to have the same environment name if they decide to open the notebook outside of github (they'll have to reset the kernel anyway).